x-pack/plugins/maps/public/components/force_refresh_checkbox.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiSwitch, EuiSwitchEvent, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  applyForceRefresh: boolean;
+  setRespondToForceRefresh: (applyGlobalTime: boolean) => void;
+  isDisabled: boolean;
+}
+
+export function ForceRefreshCheckbox({
+  applyForceRefresh,
+  setRespondToForceRefresh,
+  isDisabled,
+}: Props) {
+  const onRespondRoForceRefreshChange = (event: EuiSwitchEvent) => {
+    setRespondToForceRefresh(event.target.checked);
+  };
+
+  const tooltipContent = isDisabled
+    ? i18n.translate('xpack.maps.filterEditor.applyForceRefreshDefaultHelp', {
+        defaultMessage: `Turn this flag off for when the underlying data is static, and does not need to be refreshed based on time.`,
+      })
+    : i18n.translate('xpack.maps.filterEditor.applyForceRefreshDisabledHelp', {
+        defaultMessage: `This layer will not respond to any changes in the global query state.`,
+      });
+
+  return (
+    <EuiFormRow display="columnCompressedSwitch">
+      <EuiToolTip position="top" content={tooltipContent}>
+        <EuiSwitch
+          label={i18n.translate('xpack.maps.filterEditor.applyForceRefreshLabel', {
+            defaultMessage: `Refresh layer on global refresh`,
```

Comment:
The labels for applyTime and applyQuery are "Apply global filter to layer data" and "Apply global time to layer data". How about changing label to "Apply global refresh to layer data" for consistenty? cc @gchaps